packages/whale-api-client/src/api/rawtx.ts

```diff
@@ -27,6 +27,16 @@ export class Rawtx {
   async test (rawTx: RawTxReq): Promise<void> {
     return await this.client.requestData('POST', 'rawtx/test', rawTx)
   }
+
+  /**
+   * Get a raw transaction
+   *
+   * @param {string} txid of transaction to query
+   * @returns {Promise<string>} rawTx
+   */
+  async get (txid: string): Promise<string> {
```

Comment:
Can we include a test for this in `packages/whale-api-client/__tests__/api/rawtx.test.ts`?

Reply:
I haved added a test for that